NumLib/ODESolver/ConvergenceCriterionPerComponentResidual.h

```diff
@@ -33,11 +33,11 @@ class ConvergenceCriterionPerComponentResidual
         std::vector<double>&& relative_tolerances,
         MathLib::VecNormType norm_type);
 
-    bool hasDeltaXCheck() const override { return false; }
+    bool hasDeltaXCheck() const override { return true; }
```

Comment:
I think there should be at least documentation that there will only be diagnostic output and no actual check.